libs/journeys/ui/src/libs/MessageChatIcon/MessageChatIcon.spec.tsx

```diff
@@ -47,4 +47,114 @@ describe('MessageChatIcon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.send1} />)
+    expect(screen.getByTestId('Send1Icon')).toBeInTheDocument()
+  })
+
+  it('should return send2 icon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.send2} />)
+    expect(screen.getByTestId('Send2Icon')).toBeInTheDocument()
+  })
+
+  it('should return messageChat2 icon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.messageChat2} />)
+    expect(screen.getByTestId('MessageChat2Icon')).toBeInTheDocument()
+  })
+
+  it('should return messageCircle icon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.messageCircle} />)
+    expect(screen.getByTestId('MessageCircleIcon')).toBeInTheDocument()
+  })
+
+  it('should return messageNotifyCircle icon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.messageNotifyCircle} />)
+    expect(screen.getByTestId('MessageNotifyCircleIcon')).toBeInTheDocument()
+  })
+
+  it('should return messageNotifySquare icon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.messageNotifySquare} />)
+    expect(screen.getByTestId('MessageNotifySquareIcon')).toBeInTheDocument()
+  })
+
+  it('should return messageSquare icon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.messageSquare} />)
+    expect(screen.getByTestId('MessageSquareIcon')).toBeInTheDocument()
+  })
+
+  it('should return mail1 icon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.mail1} />)
+    expect(screen.getByTestId('Mail1Icon')).toBeInTheDocument()
+  })
+
+  it('should return linkExternal icon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.linkExternal} />)
+    expect(screen.getByTestId('LinkExternalIcon')).toBeInTheDocument()
+  })
+
+  it('should return home3 icon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.home3} />)
+    expect(screen.getByTestId('Home3Icon')).toBeInTheDocument()
+  })
+
+  it('should return home4 icon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.home4} />)
+    expect(screen.getByTestId('Home4Icon')).toBeInTheDocument()
+  })
+
+  it('should return helpCircleContained icon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.helpCircleContained} />)
+    expect(screen.getByTestId('HelpCircleContainedIcon')).toBeInTheDocument()
+  })
+
+  it('should return helpCircleSquare icon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.helpSquareContained} />)
+    expect(screen.getByTestId('HelpSquareContainedIcon')).toBeInTheDocument()
+  })
+
+  it('should return shieldCheck icon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.shieldCheck} />)
+    expect(screen.getByTestId('ShieldCheckIcon')).toBeInTheDocument()
+  })
+
+  it('should return menu1 icon', () => {
+    render(<MessageChatIcon platform={MessagePlatform.menu1} />)
+    expect(screen.getByTestId('Menu1Icon')).toBeInTheDocument()
+  })
+
+  it('should return menu icon', () => {
```

Comment:
```suggestion
  it('should return menu icon', () => {
```

Change test names to match the icons you're testing for